doc/engineering-reference/src/loop-equipment-sizing-and-other-design-data/component-sizing.tex

```diff
@@ -529,14 +565,16 @@ \subsubsection{Rated Total Cooling Capacity}\label{rated-total-cooling-capacity}
 The enthalpy of the entering air is then compared with the enthalpy of the exiting air. The calculations for air enthalpy are identical to that done for \emph{Coil:Cooling:Water}. If the entering air enthalpy is less than the exiting air enthalpy, a reference value of 48,000 J/kg is used as the entering air enthalpy. If the TotCapTempModFac calculation above yields 0 as the result, a value of 1 is used in the following calculation. If the design air mass flow rate is determined to be less than a very small flow value (0.001 kg/s) or the capacity calculated here is less than 0, the coil total cooling capacity is set equal to 0.
 
 \begin{equation}
-  \dot{Q}_{coil,des,total}   = \left\{
-                                  \begin{array}{cl}
-                                    \frac{\dot{m}_{air,des}\PB{H_{in}-H_{out}}}{TotCapTempModFac}  & if H_{in}>H_{out} \\
-                                    \frac{\dot{m}_{air,des}\PB{48000 -H_{out}}}{TotCapTempModFac}  & otherwise
-                                  \end{array}
-                                \right.
+  \dot{Q}_{coil,des,total}   = \frac{\dot{m}_{air,des}\PB{H_{in}-H_{out}}}{TotCapTempModFac} + \dot{Q}_{fan,heat,des}
```

Comment:
The Coil:Cooling:WaterToAirHeatPump:EquationFit model doesn't check for Hin>Hout, the variable speed model does. So I corrected the doc. I could have corrected the code but I never did like that test. If Hin<Hout then there should be 0 capacity.